include/sc2api/sc2_typeenums.h

```diff
@@ -456,6 +456,7 @@ namespace sc2 {
         EFFECT_VOIDRAYPRISMATICALIGNMENT = 2393,  // Target: None.
         EFFECT_WIDOWMINEATTACK = 2099,  // Target: Unit.
         EFFECT_YAMATOGUN = 401,   // Target: Unit.
+        GENERAL_MOVE = 3794,  // Target: Point.
```

Comment:
If you define GENERAL_MOVE it is ok to keep it as 3794, but you should use the old value for the MOVE command.